package.json

```diff
@@ -45,4 +45,4 @@
       "e/web/*"
     ]
   }
-}
+}
```

Comment:
your editor seems to be removing newlines at the end of files. Can you fix that do it doesn't do that? Every line should end with a newline, including the last (see https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)